packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -40,12 +44,14 @@ const ActionButton = ({
     <button
       type="button"
       data-h2-border="base(none)"
+      data-h2-radius="base(50%)"
       data-h2-cursor="base(pointer)"
       data-h2-display="base(flex)"
       data-h2-align-items="base(center)"
       data-h2-padding="base(x.5)"
       data-h2-background-color="base(foreground) base:hover(gray.lightest) base:focus(focus)"
       data-h2-transition="base:children[svg](transform 200ms ease)"
+      data-h2-font-weight="base(700)"
       {...(disabled
         ? { disabled: true, "data-h2-color": "base(black.lightest)" }
```

Comment:
We may also want to move the background color rules here. It looks a little odd when hovering disabled items due to the shape.

![Screenshot 2023-10-23 104118](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/cdbf4369-8816-47f7-80f5-b1019438554f)

Reply:
Nice catch. Fixed by [83cffcb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/83cffcb65189d8cfbe5e36e86e62495e1e28afc2)